lib/functions/failunsupportedassets.js

```diff
@@ -19,6 +19,18 @@ const { AsyncGeneratorFunction } = require("../generator/function");
  * Filter out failed assets
  */
 class FailUnsupportedAssets extends AsyncGeneratorFunction {
+    /**
+     * Construct the FailUnsupportedAssets function.
+     *
+     * @param {AEMInitiateUploadOptions} [options] Options to configure supported assets, optional
+     */
+    constructor(options) {
+        super();
+        if (options) {
```

Comment:
Do you think it would be worthwhile to default `this.options` to `{}`? In case you have later checks like `options.something`, so it doesn't crash if it's undefined/null.

Reply:
Great idea - I incorporated this change.

Taylor